client/src/Utils/dateHelper.js

```diff
@@ -0,0 +1,17 @@
+function dateHelper(date) {
+  const weekDay = new Date(date).getDay();
+
+  const weekDayMap = {
+    0: 'Monday',
+    1: 'Tuesday',
+    2: 'Wednesday',
+    3: 'Thursday',
+    4: 'Friday',
+    5: 'Saturday',
+    6: 'Sunday'
+  };
+
+  return weekDayMap[weekDay];
```

Comment:
function can be simplified and implemented with no hardcoded array

Reply:
Done, refactored.